doc/user_doc/vic_app_dev/container_operations.md

```diff
@@ -103,6 +104,39 @@ For more information about volume operations with vSphere Integrated Containers
 |`volume ls`|[List volumes](https://docs.docker.com/engine/reference/commandline/volume_ls/)|Yes. `--filter` is not supported.|
 |`volume rm`|[Remove or delete a volume](https://docs.docker.com/engine/reference/commandline/volume_rm/)|Yes|
 
+<a name="compose"></a>
+## Docker Compose Commands ##
+
+For more information about using Docker Compose with vSphere Integrated Containers Engine, see [Creating a Containerized Application with vSphere Integrated Containers Engine](creating_containerized_app_with_vic.md).
+
+| **Command** | **Docker Reference** | **Supported** |
+| --- | --- | --- |
+| `build`  | [Build or rebuild service](https://docs.docker.com/compose/reference/build/)  | No. Depends on `docker build`.|
+| `bundle`  | [Generate a Distributed Application Bundle (DAB) from the Compose file](https://docs.docker.com/compose/reference/bundle/)| No. Depends on `RepoDigests` from `docker inspect {image}`.|
```

Comment:
For the bundle support, I don't think we need to provide such details in this doc. Maybe just say "No. See issue#3648" and add a link to the issue https://github.com/vmware/vic/issues/3648